homeassistant/components/intellifire/sensor.py

```diff
@@ -21,108 +19,122 @@
 from homeassistant.util.dt import utcnow
 
 from .const import DOMAIN
-from .coordinator import IntellifireDataUpdateCoordinator
+from .coordinator import IntelliFireDataUpdateCoordinator
 from .entity import IntellifireEntity
 
 
 @dataclass(frozen=True)
-class IntellifireSensorRequiredKeysMixin:
+class IntelliFireSensorRequiredKeysMixin:
     """Mixin for required keys."""
 
-    value_fn: Callable[[IntellifirePollData], int | str | datetime | None]
+    value_fn: Callable[
+        [IntelliFireDataUpdateCoordinator], int | str | datetime | float | None
+    ]
 
 
 @dataclass(frozen=True)
-class IntellifireSensorEntityDescription(
+class IntelliFireSensorEntityDescription(
     SensorEntityDescription,
-    IntellifireSensorRequiredKeysMixin,
+    IntelliFireSensorRequiredKeysMixin,
```

Comment:
Can we do this renaming in a different PR as well?

Reply:
Yes ... reverting.